shared/utility/math/filter/YawFilter.hpp

```diff
@@ -60,9 +60,14 @@ namespace utility {
                 /// @brief Previous kinematic yaw for bias estimation
                 Scalar previous_kinematic_yaw = 0.0;
 
+                /// @brief Maximum allowed bias
+                Scalar max_bias = 0.1;
+
             public:
                 /// @brief Constructor
-                YawFilter(const Scalar alpha = 0.1, const Scalar beta = 0.01) : alpha(alpha), beta(beta) {}
+                YawFilter(const Scalar alpha = 0.1, const Scalar beta = 0.01, const Scalar max_bias = 0.1)
```

Comment:
```suggestion
                YawFilter(const Scalar alpha = 0.0, const Scalar beta = 0.0, const Scalar max_bias = 0.0)
```
or require them. should all be set from config in the module using it